py-polars/polars/lazyframe/frame.py

```diff
@@ -2599,6 +2605,55 @@ def fetch(
         """
         Collect a small number of rows for debugging purposes.
 
```

Comment:
This needs a Sphinx deprecation admonition in the docstring.

Reply:
Done